lib/session.js

```diff
@@ -30,18 +28,14 @@ module.exports = class Session {
     this[secretKey] = secret
 
     this.cookie = new Cookie(cookieOpts)
-    this[maxAge] = cookieOpts.maxAge
-    this.touch()
-    if (!this.sessionId) {
-      this.sessionId = this[generateId](this[requestKey])
-    }
-    this.encryptedSessionId = this[sign]()
+    this.sessionId = sessionId
+    this.encryptedSessionId = cookieSignature.sign(sessionId, secret)
```

Comment:
I didn't make a `sessionId` or `encryptedSessionid` symbol because fastifySession.ts needs access to it anyway.